### ISSUE TITLE
Voiceover missing in the second content card

### ISSUE THREAD

**Issue by ashengit2025** [oppia/lessons-team#1845]
Voiceover missing for the below:

Fractions->What Is A Fraction?
Fractions->Fractions of a Group
Fractions->Mixed Numbers and Number Line 2 (Intermittent Issue)
Percentages -> What is a Percentage?
Percentages -> Writing Fractions as Percentages
Percentages -> Finding a Percentage of a Number
Percentages -> Taxes and Discounts
Percentages -> Percentage Changes over Time

No voiceover for the below as well and a separate ticket has been created - https://github.com/oppia/lessons-team/issues/1841. Trying to consolidate everything into one.

Place Values 
Addition and Subtraction
Multiplication
Ratios and Proportional Reasoning

Click on the blue button to Continue
On the next content card, the voiceover seems to be disabled and when you hover over it the message says - Not available in English(United States) but the voiceover in the first content card works fine in English
See screenshot

<img width="1413" height="365" alt="Image" src="https://github.com/user-attachments/assets/f7d55b2a-2364-450a-9adc-0a246274b2d8" />
